cypress/e2e/steg1.cy.ts

```diff
@@ -0,0 +1,40 @@
+describe('Steg1-test', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/send-endringsmelding');
+  });
+  it('Sjekker at tekst laster fra Sanity', () => {
+    cy.get('.navds-heading--small').contains('Send endringer');
```

Comment:
Hva skjer hvis man har flere small heading på siden nå? Kan det bli et problem når man bruker .navds... som i stede for en data-testid? Gjelder for så vidt også feks .navds-label osv i denne filen

Reply:
^^Dette bør vi få diskutert hvordan vi vil løse - pga oppbygningen av komponentene til Nav, hindres man i å sette test-id på spesifikke komponenter, og må isåfall sette en div eller span rundt som kun er der for å plassere test-id. Må se på hvilken av disse løsningene vi vil ha

Reply:
Kan denne oppdateres nå som man kan sende inn id som prop